### PR TITLE
chore: add orb

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: orb/kustomize/orb/base/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+resources:
+  - deployment.yml
+  - service.yml
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+  - name: orb
+    newName: ghcr.io/trustbloc-cicd/orb
+    newTag: 0.1.6-snapshot-c138c96
```
Comment (troyronda) [inline orb/kustomize/orb/base/kustomization.yaml:45]: This is not the latest.

Comment (troyronda) [inline orb/kustomize/orb/base/kustomization.yaml:45]: This is not the latest.

Comment (troyronda) [inline orb/kustomize/orb/base/kustomization.yaml:45]: This is not the latest.

Comment (fqutishat) [reply orb/kustomize/orb/base/kustomization.yaml:45]: yes I will updated in next PR

**Review by rolsonquadras — APPROVED** [review]

**Review by bstasyszyn — APPROVED** [review]

**Review by sandrask — APPROVED** [review]

**Pull request merged**